staging/src/k8s.io/api/extensions/v1beta1/types.go

```diff
@@ -224,7 +224,7 @@ type RollingUpdateDeployment struct {
 	// Value can be an absolute number (ex: 5) or a percentage of desired pods (ex: 10%).
 	// This can not be 0 if MaxUnavailable is 0.
 	// Absolute number is calculated from percentage by rounding up.
-	// By default, a value of 1 is used.
+	// By default, a value of 25% is used.
```

Comment:
revert this change as well:
https://github.com/kubernetes/kubernetes/blob/0c0408c790c64d9105a85ae702c73d0714bdb850/pkg/apis/extensions/v1beta1/defaults.go#L109-L113